content/foundation/working-groups/working_groups.toml

```diff
@@ -0,0 +1,14 @@
+[[working_groups]]
+name = "Website Working Group"
+summary = "Editoral and technical oversight of the main websites and socials"
+description = """
+The website working group is responsible for the editorial and technical oversight of the main Matrix websites and social media channels.
+This includes the main matrix.org website, Event website, and the various social media channels.
+The group is responsible for ensuring that the websites are up to date, accurate, and reflect the current state of the Matrix ecosystem.
+The group is not responsible for domain-specific content of other Teams like the spec.matrix.org website, matrix.org Homeserver related configuraions or the legal and security sections of the website.
+
+Note that this is not the complete list of responsibilities, but a high level overview.
```

Comment:
we should eventually link to a published charter.

Reply:
I can prepare a button for that just like it exists kind of in the centos reference we used here :)

Reply:
![grafik](https://github.com/user-attachments/assets/a2070d64-730d-470d-9e43-68dd370bd96c)

Added it like this now. In the toml `charter = "https://somewhere"` can be added and it shows up.

I will figure out an icon for that later. Maybe today but maybe in a followup. Focusing on the other comments for now